### PR TITLE
feat(monitoring): Add support for scheduled but not yet published gro…

### PR DESCRIPTION
**Pull request by ioanpocol**
…up in monitoring (SDNTB-88)

### CODE REVIEW

**Comment by petrjasek** [comment]
imo these needs some refactoring, having to add all those ifs seems wrong

File: spec/monitoring_spec.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ var authoring = require('./helpers/authoring'),
     dashboard = require('./helpers/dashboard'),
     desks = require('./helpers/desks');
 
-describe('monitoring', function() {
+fdescribe('monitoring', function() {
```
Comment (mdhaman) [inline spec/monitoring_spec.js]: it should be `describe`.

Comment (petrjasek) [inline spec/monitoring_spec.js]: right, I was wondering why it was so fast on travis :smile:

**Comment by ioanpocol** [comment]
@petrjasek A small refactoring is done.

**Comment by petrjasek** [comment]
:+1:

**Comment by pavlovicnemanja** [comment]
:+1:

**Pull request merged**